populate_cip_20.sql

```diff
@@ -62,58 +62,58 @@ TRUNCATE settlement_scores;
 TRUNCATE settlement_observations;
 
 INSERT INTO settlements (block_number, log_index, solver, tx_hash, tx_from, tx_nonce)
-VALUES (1, 0, '\x5111'::bytea, '\x7111'::bytea, '\x5111'::bytea, 1),
-       (2, 0, '\x5222'::bytea, '\x7222'::bytea, '\x5222'::bytea, 1),
-       (5, 0, '\x5111'::bytea, '\x7333'::bytea, '\x5111'::bytea, 2),
+VALUES (1, 0, '\x5111111111111111111111111111111111111111'::bytea, '\x7111'::bytea, '\x5111111111111111111111111111111111111111'::bytea, 1),
```

Comment:
Had to change this because `Address` class (used in transfers) validates the addresses are valid.